### ISSUE TITLE
As an admin, I can update a user for an organization

### ISSUE THREAD

**Issue by CraigJZ** [rubyforgood/abalone#343]
- [ ] The UsersController `edit` and `update` actions are added
- [ ] A new view is added that renders a form, pre-populated with the data for the existing User record, for the admin to edit. Ideally this could re-use the markup for the form created in #341 
- [ ] This form can be submitted to update the user record in the database.
- [ ] A feature test is added that seeds the database with a user record, loads this page, makes an edit to that record, and submits the form
- [ ] Add specs that exercise the different outcomes for an admin and a user trying to perform this action. 

Some other considerations are that users should be managing their own passwords. One approach to this would be to give the current_user the ability to edit their own user record. This can be addressed here or split into a separate issue.

This task is dependent on #339
